languages/csharp/exercises/concept/user-defined-exceptions/.meta/Example.cs

```diff
@@ -0,0 +1,67 @@
+using System;
```

Comment:
I think the Example looks very nice now! What do you think?

Let me know when you'd like a proper review of the other files.

Reply:
> I think the Example looks very nice now!

Maybe.  I have neither perspective on this nor talent in this area.  I'm probably too hung up on the pedagogy.